packages/playout-gateway/src/tsrHandler.ts

```diff
@@ -639,6 +674,7 @@ export class TSRHandler {
 		return deviceOptions.debug || this._coreHandler.logDebug || false
 	}
 	private async _reportResult(resultsOrTimeout: UpdateDeviceOperationsResult): Promise<void> {
+		this.logger.warn(JSON.stringify(resultsOrTimeout))
```

Comment:
This seems it would be a more usefull message if there was a small description along side this

Reply:
That is true and is it really necessary to have this log statement?

Reply:
To me this looks like a leftover from debugging the code